packages/cspell-trie/src/app.ts

```diff
@@ -1,5 +1,3 @@
-#!/usr/bin/env node --max_old_space_size=8192
```

Comment:
Why remove these? They are very useful.

Reply:
Wasn't a manual change. I'm guessing from https://github.com/mysticatea/eslint-plugin-node/blob/master/docs/rules/shebang.md that it does it for anything that isn't the "bin" value

Reply:
Let's not use that rule.

Reply:
This line can go, but the `./bin.js` ones should stay.